Source/Core/DolphinQt/Debugger/CodeViewWidget.cpp

```diff
@@ -826,6 +827,31 @@ void CodeViewWidget::OnCopyCode()
   QApplication::clipboard()->setText(QString::fromStdString(text));
 }
 
+void CodeViewWidget::OnCopyWholeLine()
+{
+  const u32 addr = GetContextAddress();
+
+  const std::string textCode = [this, addr] {
+    Core::CPUThreadGuard guard(m_system);
+    return m_system.GetPowerPC().GetDebugInterface().Disassemble(&guard, addr);
+  }();
+
+  std::string wholeLineText = std::format("{:08x} {}", addr, textCode);
+
+  if (IsInstructionLoadStore(textCode))
+  {
+    const std::optional<u32> target_addr =
+        m_system.GetPowerPC().GetDebugInterface().GetMemoryAddressFromInstruction(textCode);
+
+    if (target_addr)
+    {
+      wholeLineText += std::format(" targetting {:08x}", *target_addr);
+    }
+  }
+
+  QApplication::clipboard()->setText(QString::fromStdString(wholeLineText));
```

Comment:
The following should fix your issue as we're using fmt's format instead of std.
```suggestion
  const u32 addr = GetContextAddress();

  const std::string text_code = [this, addr] {
    Core::CPUThreadGuard guard(m_system);
    return m_system.GetPowerPC().GetDebugInterface().Disassemble(&guard, addr);
  }();

  std::string whole_line = std::format("{:08x} {}", addr, text_code);

  if (IsInstructionLoadStore(text_code))
  {
    const std::optional<u32> target_addr =
        m_system.GetPowerPC().GetDebugInterface().GetMemoryAddressFromInstruction(text_code);

    if (target_addr)
    {
      whole_line += fmt::format(" targetting {:08x}", *target_addr);
    }
  }

  QApplication::clipboard()->setText(QString::fromStdString(whole_line));
```